src/core_plugins/kibana/public/visualize/wizard/step_1.html

```diff
@@ -48,6 +48,7 @@ <h2 class="kuiSubTitle kuiVerticalRhythm">
           ng-href="{{ getVisTypeUrl(type) }}"
           tooltip="{{ getVisTypeTooltip(type) }}"
           tooltip-placement="{{ getVisTypeTooltipPosition($parent.$index) }}"
+          aria-describedby="visDescription_{{::type.name}}"
```

Comment:
Replaced it now with `_.camelCase` which should be fine for now. Care for another review? :)